modules/cluster-logging-rn-5.4.13.adoc

```diff
@@ -0,0 +1,18 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-13_{context}"]
+= Logging 5.4.13
+This release includes link:https://access.redhat.com/errata/RHBA-2023:1312[OpenShift Logging Bug Fix Release 5.4.13].
```

Comment:
The link is not working, showing a 404 page not found error.

Reply:
Errata links don't work before the release goes live, this is as expected. :)